docs/orchestration/faq/upgrade_environments.md

```diff
@@ -1,8 +1,8 @@
 # Upgrading Environments to RunConfig
 
 Prefect 0.14.0 included a new Flow configuration system based on
-[RunConfig](./run_configs.md) objects. This replaces the previous system based
-on [Environment](/orchestration/execution/overview.md) objects, with
+[RunConfig](../flow_config/run_configs.md) objects. This replaces the previous system based
+on [Environment](/docs/orchestration/execution/overview.md) objects, with
```

Comment:
The Environment link works fine in the published docs, but this change breaks the link. I'd remove that part of the change.